tests/Makefile

```
@@ -0,0 +1,16 @@
+COQC=coqc -native-compiler yes
+
+all: success output
+
+success: $(addsuffix o,$(wildcard success/*.v))
+
+output: $(addsuffix o,$(wildcard output/*.v))
+
+success/%.vo: success/%.v
+	$(COQC) $<
+
+output/%.vo: output/%.v
+	input=$<; \
+	output=$${input%.v}$*.out.real; \
```

Sorry for that other nitpick, but I believe this would be enough? (and simpler)
```suggestion
	output=output/$*.out.real; \
```
because IIRC `$*` will take the value of the `%`? while in the current code, you'd get `output=output/NumbersSyntaxNumbersSyntax.out.real`

---

but this is not the only solution.

---

maybe this is simpler?

```
output/%.vo: output/%.v
	output=output/$*.out.real; \
	$(COQC) $< 2>&1 > $$output; \
	diff --strip-trailing-cr output/$*.out $$output
```

---

Indeed, that $* shouldn't be there, thanks!